src/case-swapping.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/**
+ * Implement caseSwapping function:
+ *
+ * Given a string, swap the case for each of the letters.
+ *
+ * Examples:
+ * swap "aBc"      `shouldBe` "AbC"
+ * swap "ABC"      `shouldBe` "abc"
+ * swap "123235"   `shouldBe` "123235"
+ *
+ * @param {string} phrase
+ *
+ * @return {string}
+ */
+function caseSwapping(phrase) {
```

Comment:
function names typically start with the verb https://mate-academy.github.io/style-guides/javascript.html#naming--camelCase

swapCase